test/language/class/class-subclass-binding.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    class subclass binding
+---*/
+class Base {
+  constructor(x, y) {
+    this.x = x;
+    this.y = y;
+  }
+}
+
+var obj = {};
+class Subclass extends Base {
+  constructor(x, y) {
+    super(x,y);
+    assert.sameValue(this !== obj, true);
```

Comment:
It looks messy and confusing when there are assertions mixed together, eg: 

```js
assert(descr.configurable);
assert.sameValue(descr.enumerable, false);
assert(descr.writable);
assert.sameValue(typeof descr.value, 'function');
assert.sameValue('prototype' in descr.value, false);
```

When transforming it to: 

```js
assert.sameValue(descr.configurable, true);
assert.sameValue(descr.enumerable, false);
assert.sameValue(descr.writable, true);
assert.sameValue(typeof descr.value, 'function');
assert.sameValue('prototype' in descr.value, false);
```

Has a readable uniformity to it.